data/driver_atmos_input_gfsv16.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/sh --login
+
+# GFSv16 retrospective tests: https://docs.google.com/presentation/d/1OxT0UUNhFBdiRA04alMUMOGEffmJ5BSWUwm9jtcF880/edit?usp=sharing
+
+# Usage
+if [ $# -lt 1 ]; then
+  echo "Driver script to submit the job_atmos_input.sh jobs to pull the HAFS needed GFSv16 input data."
+  echo "Usage: $0 startcycle [endcycle]"
+  echo "       $0 yyyymmddhh [yyyymmddhh]"
+  echo "e.g.:  $0 2020090100"
+  echo "or:    $0 2020090100 2020090218"
+  exit
+fi
+
+set -x
+
+if [[ -e /lfs/h2/emc ]]; then
+ #WCOSS2
+# module load ips/18.0.1.163
+ module load prod_util/2.0.13
+ BSUB="qsub"
+# cd /lfs/h2/emc/hafstemp/hafs-input/COMGFSv16
+elif [[ -e /scratch1/NCEPDEV/hwrf ]]; then
+ #Hera
+ module use /scratch2/NCEPDEV/nwprod/NCEPLIBS/modulefiles
```

Comment:
On my 1st attempt to run this, it didn't work until I added a 
module purge
before the module use...
It might make sense to add that above lines 25 and 32?

Reply:
The module commands are gone, except for these:

```
module purge
module load hpss
```